package.json

```diff
@@ -31,6 +31,7 @@
     "@headlessui/tailwindcss": "^0.2.0",
     "@heroicons/react": "^2.0.18",
     "@keystone-6/document-renderer": "^1.1.2",
+    "@storybook/client-api": "^7.6.17",
```

Comment:
Is this necessary? If not let's remove it. If it is, we can probably put it under `devDependencies`.

Reply:
This import is necessary for the storybook component handleOpen and HandleClose functions to update the arguments state inside of storybook when opening or closing the modal, via the open modal button, or ModalOverlay click event. 

https://github.com/user-attachments/assets/e3de66b7-ca85-4eab-9760-2ba5da6f6dc1

@tomershvueli